barman/cli.py

```diff
@@ -899,6 +899,18 @@ def rebuild_xlogdb(args):
             help="The name of the AWS region containing the EC2 VM and storage "
             "volumes for recovery of a snapshot backup",
         ),
+        argument(
```

Comment:
Maybe we could put this argument near `--recovery-staging-path`. They serve similar purposes. Currently they are shown like this in the console:

```console
  --recovery-staging-path RECOVERY_STAGING_PATH
                        A path to a location on the recovery host where compressed backup files will be staged during the recovery. This location must have enough available space to temporarily hold the full compressed backup. This option is *required*
                        when recovering from a compressed backup.
  --recovery-conf-filename RECOVERY_CONF_FILENAME
                        Name of the file to which recovery configuration options will be added for PostgreSQL 12 and later (default: postgresql.auto.conf).
  --snapshot-recovery-instance SNAPSHOT_RECOVERY_INSTANCE
                        Instance where the disks recovered from the snapshots are attached
  --snapshot-recovery-zone SNAPSHOT_RECOVERY_ZONE
                        Zone containing the instance and disks for the snapshot recovery (deprecated: replaced by --gcp-zone)
  --gcp-zone GCP_ZONE   Zone containing the instance and disks for the snapshot recovery
  --azure-resource-group AZURE_RESOURCE_GROUP
                        Azure resource group containing the instance and disks for recovery of a snapshot backup
  --aws-region AWS_REGION
                        The name of the AWS region containing the EC2 VM and storage volumes for recovery of a snapshot backup
  --local-staging-path LOCAL_STAGING_PATH
                        A path to a location on the local host where incremental backups will be combined during the recovery. This location must have enough available space to temporarily hold the new synthetic backup. This option is *required* when
                        recovering from an incremental backup.
```

I think it would be better if they are put near each other.